src/GraphClimber/ExpressionCompiler/Extensions/ExpressionExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace GraphClimber.ExpressionCompiler.Extensions
+{
+    /// <summary>
+    /// Provides extension methods to create
+    /// expressions easier.
+    /// </summary>
+    public static class ExpressionExtensions
+    {
+
+        /// <summary>
+        /// Expression of Null.
+        /// </summary>
+        public static readonly ConstantExpression Null = Expression.Constant(null);
+
+        /// <summary>
+        /// Creates an <see cref="ConstantExpression"/>
+        /// with the given value
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public static Expression Constant<T>(this T value)
```

Comment:
Or, perhaps, you can make this method internal. This one really spam the intellisense.